tdrs-backend/tdpservice/parsers/validators.py

```diff
@@ -552,6 +552,20 @@ def validate_header_section_matches_submission(datafile, section, generate_error
 
     return is_valid, error
 
+def validate_update_indicator(update_indicator, generate_error):
```

Comment:
is a custom function necessary here, or could we use the existing header validation? `matches` as a field validator is already present for the update indicator in `schema_defs/header.py` and, if i'm reading this correctly, any errors generated as part of `schema_defs.header.parse_and_validate()` will be logged as pre-check errors, kill parsing, and reject the file.

Reply:
Oh nice catch! The only required update to satisfy the precheck error generation would be updating the existing validator on the field to only accept a `D`.  The custom validator is on me as I pointed Andy in this direction.